pkg/findprocess/findprocess.go

```diff
@@ -19,8 +19,11 @@ var ErrNotFound = errors.New("process not found")
 func FindProcess(pid int) (process *os.Process, err error) {
 	process, err = findProcess(pid)
 	if err != nil {
-		_ = process.Release()
+		releaseErr := process.Release()
 		process = nil
+		if releaseErr != nil {
+			return process, releaseErr
```

Comment:
Can we wrap the two errors? The one from `findProcess()` seems to cover an important part.

Reply:
Yes, sure. Adapted as suggested.